tests/ui_tests/test_cluster_list_page.py

```diff
@@ -638,3 +637,93 @@ def test_check_cluster_components_page_add_few_hosts_to_component(
             assert (
                 cluster_components_page.check_that_save_btn_disabled()
             ), "Save button should be disabled"
+
+
+class TestClusterConfigPage:
+    @pytest.mark.usefixtures("create_community_cluster")
+    def test_cluster_config_page_open_by_tab(self, app_fs):
+        cluster_main_page = ClusterMainPage(app_fs.driver, app_fs.adcm.url, 1).open()
```

Comment:
same as here https://github.com/arenadata/adcm/pull/1014#discussion_r693528546

Reply:
changed in TestClusterConfigPage, other in https://github.com/arenadata/adcm/pull/1014#